cmd/util/ledger/migrations/storage_v6.go

```diff
@@ -54,79 +53,6 @@ var storageMigrationV5DecMode = func() cbor.DecMode {
 	return decMode
 }()
```

Comment:
Maybe add a comment to the lines above, for `cborTagStorageReference` and `storageMigrationV5DecMode`, that they are copied from the old Cadence version, because they are not exported.

We should maybe export the CBOR tags from Cadence to make this easier in the future.